contrib_bots/bots/github_issues/github_issues.py

```diff
@@ -48,29 +48,38 @@ def usage(self):
             '''
 
     def triage_message(self, message, client):
+        # To prevent infinite loop in private message, bot will detect
+        # if the sender name is the bot name it will return false.
+        if message['type'] == 'private':
+            return client.full_name != message['sender_full_name']
+
         original_content = message['content']
-        # This next line of code is defensive, as we
-        # never want to get into an infinite loop of posting follow
-        # ups for own follow ups!
-        if message['display_recipient'] == 'issue':
-            return False
-        is_issue = original_content.startswith('@issue')
-        return is_issue
+        is_issues_called = (original_content.startswith('@issue') or
```

Comment:
I think the rename should be singular `issue`, since the bot is called with singular.